features/step_definitions/admin_steps.rb

```diff
@@ -88,6 +88,20 @@
   click_button("Update")
 end
 
+Given /^guest comments are on$/ do
```

Comment:
Would you mind writing this step definition as a string instead of a regex? I think at this point, the goal is to use regexes only for complex steps that can't be handled by cucumber expressions.